tests/acceptance/settings-page/profile-test.js

```diff
@@ -47,4 +48,19 @@ module('Acceptance | settings-page | profile-test', function (hooks) {
     assert.strictEqual(userPage.githubDetails.username, 'rohitpaulk');
     assert.strictEqual(userPage.githubDetails.link, 'https://github.com/rohitpaulk');
   });
+
+  test('can refresh github username', async function (assert) {
+    testScenario(this.server);
+    signInAsSubscriber(this.owner, this.server);
+
+    let currentUser = this.server.schema.users.first();
+
+    assert.strictEqual(currentUser.username, 'rohitpaulk');
+
+    await profilePage.visit();
+    await profilePage.refreshFromGitHubButton.click();
+    await settled();
```

Comment:
This shouldn't be required, `click()` should already be calling `await settled()`. @libmartinito are you sure that this fails without a call to `settled()`?

Reply:
yep, works without the additional call to settled